LocationRow/LocationRow.swift

```diff
@@ -57,28 +57,27 @@ public final class LocationRow: Row<LocationCell>, RowType, PresenterRowType {
     public override func customDidSelect() {
         super.customDidSelect()
 
-        guard let controller = makeController() else { return }
-
-        presentationMode = PresentationMode.show(controllerProvider: ControllerProvider.callback {
-            controller
-        }, onDismiss: { viewController in
-            _ = viewController.navigationController?.popViewController(animated: true)
+        presentationMode = PresentationMode.presentModally(controllerProvider: ControllerProvider.callback {
```

Comment:
we should check if the row is disabled. `if !isDisabled {` or a `guard`

Reply:
Done in 53312e86a14ca57a7fe642eb43f75e63dc17e378